templates/expo-template-blank/gitignore

```diff
@@ -9,3 +9,6 @@ npm-debug.*
 *.orig.*
 web-build/
 web-report/
+
+# OSX
```

Comment:
These too, please

Reply:
3rd times a charm. My pull original pull request did not modify `expo-template-bare-minimum` or `expo-template-bare-typescript`, but those both say OSX. Do you want me to change those as well?